clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/KafkaConsumerServiceStats.java

```diff
@@ -102,6 +103,7 @@ public KafkaConsumerServiceStats(
     detectedNoRunningIngestionTopicPartitionNumSensor =
         registerSensor("detected_no_running_ingestion_topic_partition_num", new Total());
     delegateSubscribeLatencySensor = registerSensor("delegate_subscribe_latency", new Avg(), new Max());
+    waitAfterUnsubscribeLatencySensor = registerSensor("wait_after_unsubscribe_latency", new Avg(), new Max());
```

Comment:
nit/request: Can we drop a comment somewhere in here that describes that this is measuring?  The name alone is a little cryptic

Reply:
Also... What does this tells us?  Why is it useful?

Reply:
also, and I know most/all the other stats here don't do this, but It's a pattern we're hoping to correct going forward, let's include the time unit in the name of the metric.

Reply:
You bring up a good point: why is this metric needed? It turns out that there is an existing metric that tells us about the slowdown on the drainer: `consumer_records_producing_to_write_buffer_latency`. Thus, this metric is unnecessary and has been removed.